src/components/FeeSummary/FeeSummary.stories.ts

```diff
@@ -21,12 +21,29 @@ const filingData: FilingDataIF = {
   priority: false
 }
 
+const changeFilingData: FilingDataIF = {
+  filingTypeCode: FilingCodes.CHANGE_OF_REGISTRATION,
+  entityType: CorpTypeCd.SOLE_PROP,
+  priority: false
+}
+
 export const Default = Template.bind({})
 Default.args = {
   filingData: filingData,
-  payApiUrl: 'https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/',
+  payApiUrl: 'https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1/',
```

Comment:
Nice catch.

Are there any other URLs in shared components that should be updated?

Reply:
It doesn't appear so, nothing related to the migration to the silver cluster at least. 
The only other urls i could find related to github references.

Reply:
O, one in a unit test.. although it's relevance is questionable 😁 
Updated